qemu/tests/cfg/balloon_service.cfg

```diff
@@ -67,3 +67,43 @@
             type = balloon_sc_interrogate
             only Windows
             sc_interrogate_cmd = "sc interrogate balloonservice"
+        - win_virtio_driver_update:
```

Comment:
Suggest  to put installer related case together, still put it in win_virtio_driver_install_by_installer.cfg